### PR TITLE
Seed db

### LINKED ISSUE #87: Set Quiz data to database.
move data from front end to back end.

### PR DESCRIPTION
**Pull request by viktoriia-kosenko**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Seed data to DB
+
+Mongo Seeding CLI is needed to add quiz questions to db.
+To install Mongo Seeding CLI
+`npm install -g mongo-seeding-cli`
+
+Then run `seed --db-name quizzes --drop-database ./data-import`
+This command drops local quizzes db, creates new one ond seeds data from data-import folder.
```
Comment (bonboh) [inline readme.md]:
```suggestion
This command drops the local `quizzes` db, creates new one and seeds data from the `data-import `folder.
```

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+`seed --db-name quizzes --drop-database ./data-import`
```
Comment (bonboh) [inline readme.md]: We can remove this line since it is already in line 7

**Review by bonboh — CHANGES_REQUESTED** [review]
Looks good, just a couple of small nitpicks in README. It would be good to also move some of the content in `client/README.md` to `README.md` in another PR

File: readme.md (hunk 2 above)
Comment (viktoriia-kosenko) [reply readme.md]: Done :)

**Review by bonboh — APPROVED** [review]
Looks good to me 👍

**Pull request merged**